### PR TITLE
plot: plot the difference

### PR DESCRIPTION
**Pull request by amarts**
it compares the `data/results.csv` with current result. It prints to the output the % difference, which can be sent on email, and also outputs a graph which can be shared.

```
$ python3 plot/compare-and-plot.py -h
plot/compare-and-plot.py --result-dir DIR [--tag TAG] [--csv-file FILE]
```

### CODE REVIEW

**Comment by amarts** [comment]
@aravindavk @sac-urs

File: plot/compare-and-plot.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    load_reference_values()
+    t = tags[0]
+    dump_results(write_ops, ref_write_value, tags[0])
+    dump_results(read_ops, ref_read_value, tags[0])
+
+
+# TODO: get the destination directory through argument
+#dir = '/var/tmp/glusterperf/' + date
+result_dir = '/home/atumball/2019-02-14/'
```
Comment (sac-urs) [inline plot/compare-and-plot.py]: Current results are not stored there. Will you be copying the data over to that directory manually?

Comment (amarts) [reply plot/compare-and-plot.py]: Ah! this I used for testing purpose, Surely needs to be changed before merging. Just wanted to share this for review in general so we agree to model.

Comment (amarts) [reply plot/compare-and-plot.py]: Done

**Pull request merged**